pkg/controller/controller.go

```diff
@@ -324,11 +345,37 @@ func sendData(data *bytes.Buffer, contr Controller){
 	}
 	//servers.mux.Unlock()
 }
-func StartController(configFile string, servers []string, events [] string){
-     api, err := CreateK8sApiserverClient(configFile) 
+func validateKubeClusterFields(configFile, kubeURL, kubeServAcctToken string) bool {
+	if configFile == "" && kubeURL == "" && kubeServAcctToken == "" {
+		fmt.Println("Using inbuilt POD values field for Kube client configuration")
```

Comment:
Serviceaccount details from POD (See we can change  the comment field here and correct it)